scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml

```diff
@@ -0,0 +1,43 @@
+- job-template:
+    name: 'cloud-mkcloud{version}-job-96compute-{arch}'
+    node: cloud-trigger
+
+    triggers:
+      - timed: '0 0 * * 6'
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            ### Controller Node Configuration ###
+            controller_node_memory=31457280
+            controller_hdd_size=20
+            ### Admin Node Configuration ###
+            admin_node_memory=31457280
+            adminvcpus=10
+            adminnode_hdd_size=100 # GB
+            ### Compute Node Configuration ###
+            compute_node_memory=2097152 # KB's
+            vcpus=1 # Default 1 vcpu
+            TESTHEAD=1
+            cloudsource=develcloud{version}
+            nodenumber=96
+            # HA Cluster(s) Configuration
+            clusterconfig="data+services=3:network=3"
+            want_node_aliases='controller=3,network=3,compute=90'
+            drbd_hdd_size=100
+            hacloud=1
+            networkingplugin=openvswitch
+            networkingmode=vlan
+            want_dvr=1
+            cephvolumenumber=0
+            mkcloudtarget=all
+            label=big_machine
```

Comment:
that should probably be  a architecture specific label, e.g. label={label} with label being expanded above in the cloud-mkcloud7.yaml